src/lib/access/MergeTable.cpp

```diff
@@ -56,5 +57,34 @@ void MergeStore::executePlanOperation() {
 }
 
 std::shared_ptr<PlanOperation> MergeStore::parse(const Json::Value& data) { return std::make_shared<MergeStore>(); }
+
+namespace {
+auto _3 = QueryParser::registerPlanOperation<MergeColumnStore>("MergeColumnStore");
+}
+
+MergeColumnStore::MergeColumnStore(bool forceFullIndexRebuild) : _forceFullIndexRebuild(forceFullIndexRebuild) {};
+
+MergeColumnStore::~MergeColumnStore() {}
+
+void MergeColumnStore::executePlanOperation() {
+  auto t = checked_pointer_cast<const storage::Store>(getInputTable());
+  auto store = std::const_pointer_cast<storage::Store>(t);
+  storage::ColumnStoreMerger merger = storage::ColumnStoreMerger(store, _forceFullIndexRebuild);
+
+  if (!store->isColumnStore())
+    throw std::runtime_error("Column store merge applied to row store column!");
```

Comment:
I believe this error message should be something like "Column Store Merger applied to a hybrid or row table"